### PR TITLE
[#2992] Allow custom product name for download page.

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
---------

We now have a product name which is different than the base url name.

Changes
-----------

Allow a custom product name, other than the name used to construct the URL

How to test
--------------

reviewers: @alibotean 

this is just to let you know about the changes.

you can see the download page in the latest staging for MFT

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

File: release-notes.rst [hunk 1]
```diff
@@ -4,6 +4,12 @@ chevah-brink release notes
 Here are the release notes for past brink version.
 
 
+0.55.2 - 08/09/2015
```
Comment (alibotean) [inline release-notes.rst]: 08/09?

**Comment by alibotean** [comment]
> http://staging.sftpplus.com/downloads/mft/

Check.

**Comment by alibotean** [comment]
Just a minor comment about the version date.

changes-approved

**Pull request closed without merging**